src/rgw/rgw_motr.cc

```diff
@@ -0,0 +1,133 @@
+#include "include/compat.h"
```

Comment:
The new meta is to put store-specific code in rgw/store/<storename>  RadosStore is exempted for now, because of legacy, but the plan is to move it also, once modularization is done.

Reply:
@sanalpk as @dang suggested, can you please create dir `src/rgw/store/motr/` and move all newly created files in your PR to that directory?

Reply:
I think he suggested this once modularization is done, 
currently rgw_rados.cc & rgw_rados.h are in src/rgw, but we can do it later.

Reply:
RadosStore is exempted until modularization is done, but others has to follow the norm. Please check the [dbstore](https://github.com/ceph/ceph/tree/master/src/rgw/store/dbstore) for reference.